project_cost_report/static/src/js/report_action.js

```diff
@@ -0,0 +1,385 @@
+/*
+    © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL.html).
+*/
+odoo.define("project_cost_report.ReportAction", function (require) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['odoo' is not defined. (no-undef)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=3675949)